src/kompos/cli/terraform.py

```diff
@@ -111,6 +113,12 @@ def configure(self, parser):
                             help='for use with "apply". Proceeds with the apply without'
                                  'waiting for user confirmation.',
                             action='store_true')
+        parser.add_argument('--himl',
+                            action='store',
+                            dest='himl_args',
+                            default=None,
+                            help='for passing arguments to himl'
+                                 '--himl="--skip-interpolation-validation --skip-secrets"')
```

Comment:
Are those the only flags allowed?

Reply:
No, you can use any himl flags, just need to wrap them in quotes.

Reply:
Better not include those examples on the help string, just mentioned that any himl argument is supported.